src/print.rs

```diff
@@ -36,7 +37,59 @@ pub fn prompt(args: ArgMatches) {
         writeln!(handle).unwrap();
     }
 
-    let modules = PROMPT_ORDER
+    let mut prompt_order: Vec<&str> = Vec::new();
+    //List of default prompt order
+    let default_prompt_order: &[&str] = &[
+        "username",
+        "directory",
+        "git_branch",
+        "git_status",
+        "package",
+        "nodejs",
+        "rust",
+        "python",
+        "golang",
+        "cmd_duration",
+        "line_break",
+        "jobs",
+        "battery",
+        "character",
+    ];
+
+    // Write out a custom prompt order
+    if let Some(values) = config.get_as_array("prompt_order") {
```

Comment:
To try to keep things semantic, I think it would be best if we replaced references to `values` with `modules`.